compliance/compliance.go

```diff
@@ -2013,3 +2018,51 @@ func ForwardReferencesDisallowed(c *fluent.GRIBIClient, t testing.TB, _ ...TestO
 			AsResult(),
 		chk.IgnoreOperationID())
 }
+
+// AddNHGReferencingToUknownNH tests a case where a nexthop group references a nexthop IP that does not Resolve.
+func AddNHGReferencingToUknownNH(c *fluent.GRIBIClient, wantACK fluent.ProgrammingResult, t testing.TB, _ ...TestOpt) {
+	defer flushServer(c, t)
+	ops := []func(){
+		func() {
+			c.Modify().AddEntry(t,
+				fluent.NextHopEntry().WithNetworkInstance(defaultNetworkInstanceName).WithIndex(1).WithIPAddress("192.0.2.30"),
+				fluent.NextHopGroupEntry().WithNetworkInstance(defaultNetworkInstanceName).WithID(1).AddNextHop(1, 1))
+
+		},
+	}
+	t.Log("Sleeping for 100 seconds to allow the DUT to process the new configuration")
+	time.Sleep(100 * time.Second)
+
+	res := DoModifyOps(c, t, ops, wantACK, false)
+	chk.HasResult(t, res,
+		fluent.OperationResult().
+			WithNextHopGroupOperation(1).
+			WithOperationType(constants.Add).
+			WithProgrammingResult(wantACK).
+			AsResult(),
+		chk.IgnoreOperationID())
+
+}
+
+// AddNHGReferencingToDownPort tests a case where a nexthop group references a nexthop port that is down.
```

Comment:
Per comment above, I think this should be in featureprofiles rather than a compliance test here -- since the port being down is not something that our test framework here understands.